plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -1,3756 +1,435 @@
+            }
+        },
+        template: '<div class="crash-stacktrace bu-p-5">\
+        <div class="bu-level is-mobile">\
+        <div class="bu-level-left"><slot name="header-left"></slot></div>\
+        <div class="bu-level-right"><slot name="header-right"></slot></div>\
+        </div>\
+        <pre><span class="crash-stacktrace__line-numbers">{{lineNumbers}}</span><code ref="code" class="crash-stacktrace__code" v-html="highlightedCode"></code></pre>\
+        </div>'
+    });
 
-        $(".filter-segmentation").off("cly-select-change").on("cly-select-change", function(e, val) {
-            self.filterCrashes(val);
-        });
-        $(".action-segmentation").off("cly-select-change").on("cly-select-change", function(e, val) {
-            if (val !== "") {
-                $(".action-segmentation").clySelectSetSelection("", jQuery.i18n.map["crashes.make-action"]);
-                if (val === "crash-resolve") {
-                    CountlyHelpers.confirm(jQuery.i18n.prop("crashes.confirm-action-resolved", self.selectedCrashesIds.length), "red", function(result) {
-                        if (!result) {
-                            return true;
-                        }
-                        countlyCrashes.markResolve(self.selectedCrashesIds, function(data) {
-                            if (!data) {
-                                CountlyHelpers.alert(jQuery.i18n.map["crashes.try-later"], "red");
-                                self.resetSelection(true);
-                            }
-                        });
-                    });
-                }
-                else if (val === "crash-unresolve") {
-                    CountlyHelpers.confirm(jQuery.i18n.prop("crashes.confirm-action-unresolved", self.selectedCrashesIds.length), "red", function(result) {
-                        if (!result) {
-                            return true;
-                        }
-                        countlyCrashes.markUnresolve(self.selectedCrashesIds, function(data) {
-                            if (!data) {
-                                CountlyHelpers.alert(jQuery.i18n.map["crashes.try-later"], "red");
-                            }
-                            else {
-                                self.resetSelection(true);
-                            }
-                        });
-                    });
+    var CrashgroupView = countlyVue.views.BaseView.extend({
+        template: "#crashes-crashgroup",
+        components: {"crash-stacktrace": CrashStacktraceView},
+        data: function() {
+            return {
+                appId: countlyCommon.ACTIVE_APP_ID,
+                authToken: countlyGlobal.auth_token,
+                chartBy: "os_version",
+                diagnosticsOrder: ["ram", "disk", "battery", "running", "sessions"],
+                eventLogsEnabled: countlyGlobal.plugins.includes("crashes-event-logs"),
```

Comment:
IE11 has no "includes" support :/

Reply:
We should polyfill it, "indexof !== -1" is ugly 🥺